checker/check-request-parameters-max-items-updated.go

```diff
@@ -0,0 +1,59 @@
+package checker
+
+import (
+	"fmt"
+
+	"github.com/tufin/oasdiff/diff"
+)
+
+func RequestParameterMaxItemsUpdatedCheck(diffReport *diff.Diff, operationsSources *diff.OperationsSourcesMap, config Config) Changes {
+	result := make(Changes, 0)
+	if diffReport.PathsDiff == nil {
+		return result
+	}
+	for path, pathItem := range diffReport.PathsDiff.Modified {
+		if pathItem.OperationsDiff == nil {
+			continue
+		}
+		for operation, operationItem := range pathItem.OperationsDiff.Modified {
+			if operationItem.ParametersDiff == nil {
+				continue
+			}
+			for paramLocation, paramDiffs := range operationItem.ParametersDiff.Modified {
+				for paramName, paramDiff := range paramDiffs {
+					if paramDiff.SchemaDiff == nil {
+						continue
+					}
+					maxItemsDiff := paramDiff.SchemaDiff.ItemsDiff.MaxItemsDiff
+					if maxItemsDiff == nil {
+						continue
+					}
+					if maxItemsDiff.From == nil ||
+						maxItemsDiff.To == nil {
+						continue
+					}
```

Comment:
I'm not sure but it's possible that From==nil means no max
In that case, changing nil to a number would, in effect, be breaking

Reply:
I think we have max-items set checker for this, here i just followed same pattern as other max-item checkers